CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [1.3.3] - 2023-05-15
```

Comment:
1. Version does not comply with semantic versioning - https://semver.org/
2. Specified date.
3. Duplicated changelog points.